src/types.ts

```diff
@@ -45,3 +45,7 @@ export interface Version {
   minor?: number;
   patch?: number;
 }
+
+export type VkontakteIcon = React.HTMLAttributes<
+  SVGSVGElement | HTMLDivElement
+>;
```

Comment:
Предлагаю назвать его так, чтобы было понятно почему `SVGSVGElement | HTMLDivElement`

Что-то типа того)

```suggestion
/**
 * `getRootRef` в `@vkontakte/icons` v1 возвращает `HTMLDivElement`.
 * С v2 возвращает `SVGSVGElement`.
 */
export type VkontakteIconBackwardCompatibilityElement = HTMLDivElement | SVGSVGElement;
```

в `DropdownIcon` вернуть `React.HTMLAttributes<VkontakteIconBackwardCompatibilityElement>`

Reply:
➕ 
Сейчас еще во второй пулл перенесу.